### PR TITLE
Add dynamic partitions + seine fstab changes

### PR DESCRIPTION
**Pull request by Haxk20**
This fstab file was based on Kumano which is not
DP platform and thus they were missing.
While at it i included changes from seine fstab.
Such as:
	Add metadata partition
	Enforce oem_a
	Fix USB external storage path
	Fix userdata mount flags

### CODE REVIEW

File: rootdir/vendor/etc/fstab.edo [hunk 1]
```diff
@@ -2,9 +2,17 @@
 # The filesystem that contains the filesystem checker binary (typically /system) cannot
 # specify MF_CHECK, and must come before any filesystems that do specify MF_CHECK
 
-/dev/block/bootdevice/by-name/system       /            ext4    ro,barrier=1                                                  wait,slotselect,first_stage_mount
-/dev/block/bootdevice/by-name/oem          /odm         ext4    ro,barrier=1                                                  wait,slotselect,first_stage_mount
-/dev/block/bootdevice/by-name/userdata     /data        ext4    noatime,nosuid,nodev,barrier=1,data=ordered,noauto_da_alloc,errors=panic latemount,wait,check,formattable,fileencryption=ice,quota,reservedsize=128M
+#Logical Partitions
```
Comment (pablomh) [inline rootdir/vendor/etc/fstab.edo:5]: Add a space after the '#'.

File: rootdir/vendor/etc/fstab.edo [hunk 2]
```diff
@@ -2,9 +2,17 @@
...
+system                                                  /system                   ext4    ro,barrier=1,discard                                 wait,slotselect,avb=vbmeta_system,logical,first_stage_mount,avb_keys=/avb/q-gsi.avbpubkey
```
Comment (pablomh) [inline rootdir/vendor/etc/fstab.edo:6]: Please, align the fields as done for seine :)

File: rootdir/vendor/etc/fstab.edo [hunk 3]
```diff
@@ -2,9 +2,17 @@
...
+product                                                 /product                  ext4    ro,barrier=1,discard                                 wait,slotselect,avb=vbmeta_system,logical,first_stage_mount
+vendor                                                  /vendor                   ext4    ro,barrier=1,discard                                 wait,slotselect,avb,logical,first_stage_mount
+
+# Real first stage mount partitions
+/dev/block/by-name/metadata                /metadata             ext4     noatime,nosuid,nodev,discard                                         wait,formattable,first_stage_mount
+/dev/block/bootdevice/by-name/oem_a          /odm         ext4    ro,barrier=1                                                  wait,first_stage_mount
+
+#
```
Comment (pablomh) [inline rootdir/vendor/etc/fstab.edo:14]: 's/^#/# Other partitions/'

File: rootdir/vendor/etc/fstab.edo (hunk 2 above)
Comment (pablomh) [inline rootdir/vendor/etc/fstab.edo:6]: Are we sure about the `avb_keys=/avb/q-gsi.avbpubkey` part?

**Review by pablomh — COMMENTED** [review]
All in all, for the time being I'd just copy the `seine` one makeing that the `avb_keys` part is right.

File: rootdir/vendor/etc/fstab.edo (hunk 2 above)
Comment (ix5) [reply rootdir/vendor/etc/fstab.edo:6]: Hehe... yeah edo is definitively not going to run on Q, ever.

@pablomh C.f. https://github.com/sonyxperiadev/device-sony-common/pull/748

Comment (Haxk20) [reply rootdir/vendor/etc/fstab.edo]: Done

File: rootdir/vendor/etc/fstab.edo (hunk 3 above)
Comment (Haxk20) [reply rootdir/vendor/etc/fstab.edo]: Done

File: rootdir/vendor/etc/fstab.edo (hunk 2 above)
Comment (Haxk20) [reply rootdir/vendor/etc/fstab.edo]: Done

File: rootdir/vendor/etc/fstab.edo (hunk 1 above)
Comment (Haxk20) [reply rootdir/vendor/etc/fstab.edo]: Done

**Pull request merged**